Lib/test/test_math.py

```diff
@@ -623,6 +624,100 @@ def testFmod(self):
         self.assertEqual(math.fmod(0.0, NINF), 0.0)
         self.assertRaises(ValueError, math.fmod, INF, INF)
 
+    def test_fmax(self):
+        self.assertRaises(TypeError, math.fmax)
+        self.assertRaises(TypeError, math.fmax, 'x', 'y')
+
+        self.assertEqual(math.fmax(0., 0.), 0.)
+        self.assertEqual(math.fmax(1., 2.), 2.)
+        self.assertEqual(math.fmax(2., 1.), 2.)
+
+        # fmax() does not need to be sensitive to the sign of 0 (§F.10.9.2).
+        self.assertEqual(math.fmax(+0., -0.), 0.)
```

Comment:
Actually, this test becomes now useless because if fmax() is sensitive to the sign of 0, then the test passes (because of assertEqual) and if it's not sensitive, it also passes (as it could return +/-0 but I wouldn't know about it in advance). I'll just remove this.